README

```diff
@@ -221,7 +221,8 @@ REQUIREMENTS:
         During runtime, you need the following additional
         dependencies:
 
-        util-linux >= v2.27.1 required
+        util-linux >= v2.27.1 required (including but not limited to mount,
+                                        umount, swapon, swapoff)
```

Comment:
not just that. also "sulogin" (we require "--force" switch which busybox doesn't know)

also, the fsck tool with its "-l" option which busybox doesn't know. 

agetty...